.pubnub.yml

```diff
@@ -1,5 +1,12 @@
 --- 
 changelog:
+  - 
+    changes: 
+      - 
+        text: "Signals"
+        type: improvement
+    date: Oct 10, 19  
+    version: v4.3.0  
```

Comment:
Doesn't the version also need to be modified on the root?

Reply:
yep, updating.